R/stats_wrapper.R

```diff
@@ -169,6 +169,9 @@ do_svd.kv <- function(df,
   # this is executed on each group
   do_svd_each <- function(df){
     matrix <-simple_cast(df, subject_col, dimension_col, value_col, fun.aggregate = fun.aggregate, fill=fill)
+    if(any(is.na(matrix))){
+      stop("NA is not supported as value.")
```

Comment:
Looks like this is true for all of Subject, Key, and Value.
Would something like following make it easier to understand?
"NA is not supported as value for Subject, Key, or Value."